code/datums/skills/operating.dm

```diff
@@ -0,0 +1,5 @@
+/datum/skill/operating
```

Comment:
this is a very generic name for a very specific skill

Reply:
i cannot think of more appropriate name tho. it is a skill in operating ancient machinery and i think it is easy to get